roles/api.py

```diff
@@ -0,0 +1,23 @@
+"""
+API for roles
+"""
+from rolepermissions.verifications import has_object_permission
+
+from roles.models import Role
+
+
+def get_advance_searchable_programs(user):
```

Comment:
Honestly right now it exposes users also for not live programs, but I do not think this is a decision that should be done at this level